[2022-08-29 16:52:16,478] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: h1b_lca_data_v3.entry_task scheduled__2021-04-10T06:00:00+00:00 [queued]>
[2022-08-29 16:52:16,493] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: h1b_lca_data_v3.entry_task scheduled__2021-04-10T06:00:00+00:00 [queued]>
[2022-08-29 16:52:16,495] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-08-29 16:52:16,496] {taskinstance.py:1239} INFO - Starting attempt 44 of 44
[2022-08-29 16:52:16,497] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-08-29 16:52:16,515] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): entry_task> on 2021-04-10 06:00:00+00:00
[2022-08-29 16:52:16,524] {standard_task_runner.py:52} INFO - Started process 279 to run task
[2022-08-29 16:52:16,531] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'h1b_lca_data_v3', 'entry_task', 'scheduled__2021-04-10T06:00:00+00:00', '--job-id', '1588', '--raw', '--subdir', 'DAGS_FOLDER/upload_h1b_data_to_gcp.py', '--cfg-path', '/tmp/tmpdv9sua7j', '--error-file', '/tmp/tmpx52slu0m']
[2022-08-29 16:52:16,534] {standard_task_runner.py:77} INFO - Job 1588: Subtask entry_task
[2022-08-29 16:52:16,621] {logging_mixin.py:109} INFO - Running <TaskInstance: h1b_lca_data_v3.entry_task scheduled__2021-04-10T06:00:00+00:00 [running]> on host f153c6531160
[2022-08-29 16:52:16,700] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=h1b_lca_data_v3
AIRFLOW_CTX_TASK_ID=entry_task
AIRFLOW_CTX_EXECUTION_DATE=2021-04-10T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-04-10T06:00:00+00:00
[2022-08-29 16:52:16,702] {logging_mixin.py:109} INFO - <class 'str'>
[2022-08-29 16:52:16,704] {python.py:175} INFO - Done. Returned value was: None
[2022-08-29 16:52:16,721] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=h1b_lca_data_v3, task_id=entry_task, execution_date=20210410T060000, start_date=20220829T165216, end_date=20220829T165216
[2022-08-29 16:52:16,785] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-08-29 16:52:16,831] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
